torch/_dynamo/convert_frame.py

```diff
@@ -913,7 +914,9 @@ class FrameInfo:
 
 
 def fullgraph_capture(
-    frame: FrameInfo, _is_export_deprecated_do_not_use: bool = False
+    frame: FrameInfo,
+    _is_export_deprecated_do_not_use: bool = False,
+    _export_constraints: Optional[list[Constraint]] = None,
```

Comment:
```suggestion
    constraints: Optional[list[Constraint]] = None,
    *,
    _is_export_deprecated_do_not_use: bool = False,
```
constraints doesn't have to be export only